sys/include/sema.h

```diff
@@ -33,18 +34,27 @@ extern "C" {
 /**
  * @brief   Creates semaphore statically.
  *
- * @param[in] value Initial value for the semaphore.
+ * The value for the statically created semaphore must be 0
  *
  * @return  Statically initialized semaphore.
  */
-#define SEMA_CREATE(value)      { (value), PRIORITY_QUEUE_INIT }
+#define SEMA_CREATE()           { (0), SEMA_OK, MUTEX_INIT() }
```

Comment:
Good point, actually it can only be initialized with a value different from 0. In order to initialize with 0 I need to initialize the mutex as locked, and this requires a change in the `mutex.h` file to make it clean (see #6210).

If that get's merged fast then I'll update this PR, if this PR gets merged first, I'll create a new one :-) The only addition would be:

```C
/**
 * @brief   Creates 0 initialized semaphore statically.
 * @return  Statically initialized semaphore.
 */
#define SEMA_CREATE_EMPTY()        { (value), SEMA_OK, MUTEX_INIT_LOCKED }
```